Update pipeline to release when we tag to github and documentation for release (#53)
mattermost/mattermost-plugin-cloud: README.md

  | Mattermost E20 License | `E20 license file contents` |
8. Enable the plugin
The plugin is now usable. Make changes to your plugin and run `make deploy` again to run those changes on your local Mattermost server.

## Release

To release a new version of the `cloud plugin` please do the following steps:

- Bump the version to the desire version in the files:

>   - `plugin.json`

Reviewer: You only need to change it here then run `make apply`

Author: done

  - `webapp/src/manifest.js`
  - `server/manifest.go`

- Create a PR with the changes, see [example](https://github.com/mattermost/mattermost-plugin-cloud/pull/52)

> - After the PR is merged, generate the release notes. Here will show the commands but if need more information please refer to the [internal docs](https://app.gitbook.com/@mattermost-private/s/internal-documentation/cloud/cloud/releases/rel-notes)

Reviewer 2: ```suggestion
- After the PR is merged, generate the release notes with the commands below. For more information please refer to the [internal docs.](https://app.gitbook.com/@mattermost-private/s/internal-documentation/cloud/cloud/releases/rel-notes)
```
